src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -129,34 +129,32 @@ public void CreateApplicationMetadata(string org, string app, string appTitle)
                 Id = ApplicationHelper.GetFormattedApplicationId(org, app),
                 VersionId = null,
                 Org = org,
-
                 Created = DateTime.UtcNow,
                 CreatedBy = developer,
                 LastChanged = DateTime.UtcNow,
-                LastChangedBy = developer
+                LastChangedBy = developer,
```

Comment:
Nice! More readable than the previous code.

Reply:
you're so kind to me!